BridgeCareApp/BridgeCare/Services/SummaryReport/SummaryReportBridgeData.cs

```diff
@@ -0,0 +1,361 @@
+                coln = AddSimulationYearData(worksheet, row, coln, lastYearData, familyId);
+
+                // Add all yrs from current year simulation data
+                for (var index = 1; index < yearsData.Count(); index++)
+                {
+                    coln = AddSimulationYearData(worksheet, row, coln, yearsData[index], familyId);
+                }
+                row++;
+            }
+            currentCell.Row = row - 1;
+            currentCell.Coln = coln - 1;
+
+        }
+
+        private static int AddSimulationYearData(ExcelWorksheet worksheet, int row, int coln, YearsData yearData, string familyId)
+        {
+            worksheet.Cells[row, ++coln].Value = Convert.ToInt32(familyId) > 10 ? "N" : yearData.Deck;
+            worksheet.Cells[row, ++coln].Value = Convert.ToInt32(familyId) > 10 ? "N" : yearData.Super;
+            worksheet.Cells[row, ++coln].Value = Convert.ToInt32(familyId) > 10 ? "N" : yearData.Sub;
+            worksheet.Cells[row, ++coln].Value = Convert.ToInt32(familyId) < 11 ? "N" : yearData.Culv;
+            worksheet.Cells[row, ++coln].Value = Convert.ToInt32(familyId) > 10 ? "N" : yearData.DeckD;
+            worksheet.Cells[row, ++coln].Value = Convert.ToInt32(familyId) > 10 ? "N" : yearData.SuperD;
+            worksheet.Cells[row, ++coln].Value = Convert.ToInt32(familyId) > 10 ? "N" : yearData.SubD;
+            worksheet.Cells[row, ++coln].Value = Convert.ToInt32(familyId) < 11 ? "N" : yearData.CulvD;
+            worksheet.Cells[row, ++coln].Value = yearData.MinC;
+            worksheet.Cells[row, ++coln].Value = yearData.SD;
+            if (yearData.Year != 0)
+            {
+                worksheet.Cells[row, ++coln].Value = yearData.Project;
+                worksheet.Cells[row, ++coln].Value = yearData.Cost;
+            }
+            // Empty column
+            coln++;
+            return coln;
+        }
+
+        private static List<SimulationDataModel> GetSimulationDataModels(DataTable simulationDataTable, List<int> simulationYears, IQueryable<ReportProjectCost> projectCostModels)
+        {
+            var simulationDMs = new List<SimulationDataModel>();
+            var projectCostsList = projectCostModels.ToList();
+            foreach (DataRow simulationRow in simulationDataTable.Rows)
+            {
+                var simulationDM = CreatePrevYearSimulationMdel(simulationRow);
+                var projectCostEntry = projectCostsList.Where(pc => pc.SECTIONID == Convert.ToUInt32(simulationRow["SECTIONID"])).FirstOrDefault();
+                AddAllYearsData(simulationRow, simulationYears, projectCostEntry, simulationDM);
+                simulationDMs.Add(simulationDM);
+            }
+
+            return simulationDMs;
+        }
+
+        private static void AddAllYearsData(DataRow simulationRow, List<int> simulationYears, ReportProjectCost projectCostEntry, SimulationDataModel simulationDM)
+        {
+            var yearsDMs = new List<YearsData>();
```

Comment:
Don't abbreviate - is YearsData a data model?

Reply:
Removed abbrevation.
Yes it is DataModel which is part of Simulation Data from Report.